dockstore-webservice/src/test/java/core/WDLParseTest.java

```diff
@@ -56,7 +58,7 @@ public void testWDLMetadataExampleWithMerge() throws IOException {
         Entry entry = sInterface
             .parseWorkflowContent(new Tool(), filePath, FileUtils.readFileToString(new File(filePath), StandardCharsets.UTF_8), new HashSet<>());
         assertTrue("incorrect author", entry.getAuthor().split(",").length >= 2);
-        assertTrue("incorrect email", entry.getEmail().isEmpty());
```

Comment:
Double-check this (does the database allow both empty and null, ideally it would be just one)

Reply:
Looks like email can be null.